### ISSUE TITLE
no resolve ipv6 breaks resty-cli on nginx without ipv6 support

### ISSUE THREAD

**Issue by spacewander** [openresty/resty-cli#19]
When I run the latest version of `resty-cli`, I get:
```
nginx: [emerg] host not found in resolver "ipv6=off" in /var/folders/65/qbj3jwt50tn2sfc4lg8lgvsc0000gn/T/JrbzDuSN3v/conf/nginx.conf:58
```

After debug, I find it is caused by `resolver 114.114.114.114 ipv6=off;`.

There is an undocumented feature/bug of Nginx:
Nginx only parse `ipv6=off` as an parameter when it is built with `NGX_HAVE_INET6`:
```
#if (NGX_HAVE_INET6)
        if (ngx_strncmp(names[i].data, "ipv6=", 5) == 0) {

            if (ngx_strcmp(&names[i].data[5], "on") == 0) {
                r->ipv6 = 1;

            } else if (ngx_strcmp(&names[i].data[5], "off") == 0) {
                r->ipv6 = 0;

            } else {
                ngx_conf_log_error(NGX_LOG_EMERG, cf, 0,
                                   "invalid parameter: %V", &names[i]);
                return NULL;
            }

            continue;
        }
#endif
```
Otherwise, `ipv6=off` will be parsed as a host, and this host is invalid.

**Comment by thibaultcha** [comment]
Thanks for reporting this early!

As a short workaround if you still want to use the master branch of resty-cli, I suggest that you use the `--ipv6` flag.

I will propose a patch to prevent the setting of this flag when Nginx is not compiled with IPv6 support!

**Comment by thibaultcha** [comment]
I just proposed a fix in #20 (took a couple of tries + added some tests in a forced push later on, hence the spam, sorry).

Your feedback is much appreciated :)

**Comment by spacewander** [comment]
@thibaultcha 
This fix works for me. Thanks for your quick fix.
